src/Form/RedirectSettingsForm.php

```diff
@@ -55,6 +55,12 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#options' => redirect_status_code_options(),
       '#default_value' => $config->get('default_status_code'),
     );
+    $form['redirect_error_log'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Log 404 errors.'),
+      '#default_value' => $config->get('error_log'),
+      '#description' => t('In case you directly import 404 errors from the webserver, you need to disable logging to avoid duplicates.'),
```

Comment:
the description needs work but what exactly we want to add depends on the decisions above.